kolibri/plugins/hooks.py

```diff
@@ -337,8 +350,7 @@ def registered_hooks(self):
         hook is accessed, it's also instantiated, giving it a chance to re-do
         things that it wants done on every event.
         """
-        for HookClass in self._meta.registered_hooks:
-            yield HookClass()
+        return [HookClass() for HookClass in self._meta.registered_hooks]
```

Comment:
Would rather not change this behaviour unless there is a very good reason to prefer a list over a generator (as it touches very disparate parts of the backend code and could cause unintended consequences).